lib/puppet/rest/routes.rb

```diff
@@ -22,5 +22,15 @@ def self.get_certificate(client, name)
       end
       body
     end
+
+    def self.get_crls(client, name)
+      header = { 'Accept' => 'text/plain', 'accept-encoding' => ACCEPT_ENCODING }
+      body = ''
+      client.get("certificate_revocation_list/#{name}", header: header) do |chunk|
+        body << chunk
```

Comment:
We'll need to think about how to handle very large CRLs here at some point. This may read them in chunks, but it still ultimately loads the whole thing into memory.

Reply:
Let's do that now. Do we want to provide a block call signature, pass in an writer, or some other way?

Reply:
I added what I think would be considered a "block call signature".